OneCommunityGlobal/HighestGoodNetworkApp
# Task notification on adding and editing a task with modal differences

### Before testing
1. Pull this [PR on HGNRest](https://github.com/OneCommunityGlobal/HGNRest/pull/166)
2. Make sure to run 'npm install' and 'npm run build' on backend.
3. Pull this PR.
4. Make sure to run 'npm install' on frontend

### To test:

### Test adding task:
1. Add a task (make sure to assign it to a user in the currently logged in user's team - otherwise it shouldn't show up)
5. Go to dashboard and find user/task - it should have a red bell next to it.
6. Click on the red bell to open a modal that shows the differences

### Test editing task:
1. Edit a task (again, make sure to edit a task that belongs to a person on the team of currently logged in user)
2. Go to dashboard and find user/task - it should have a red bell next to it.
3. Click on the red bell to open a modal that shows the differences
